projects/netgrif-components/src/lib/header/header-modes/edit-mode/edit-mode.component.scss

```
@@ -5,3 +5,59 @@
+    background: transparent;
+    color: inherit;
+    cursor: pointer;
+    line-height: 0;
+
+    &:focus-visible {
+        outline: 2px solid currentColor;
+        outline-offset: 2px;
+    }
+}
+
+::ng-deep .header-field-icon-container {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)